spec/models/automated_checks/claim_verifiers/identity_spec.rb

```diff
@@ -87,767 +88,847 @@ module ClaimVerifiers
 
-              before { perform }
+                describe "identity confirmation task" do
+                  subject(:identity_confirmation_task) { claim_arg.tasks.find_by(name: "identity_confirmation") }
 
-              describe "#claim_verifier_match" do
-                subject(:claim_verifier_match) { identity_confirmation_task.claim_verifier_match }
+                  before { perform }
 
-                it { is_expected.to eq "any" }
-              end
+                  describe "#claim_verifier_match" do
+                    subject(:claim_verifier_match) { identity_confirmation_task.claim_verifier_match }
 
-              describe "#created_by" do
-                subject(:created_by) { identity_confirmation_task.created_by }
+                    it { is_expected.to eq "any" }
+                  end
 
-                it { is_expected.to eq nil }
-              end
+                  describe "#created_by" do
+                    subject(:created_by) { identity_confirmation_task.created_by }
 
-              describe "#passed" do
-                subject(:passed) { identity_confirmation_task.passed }
+                    it { is_expected.to eq nil }
+                  end
 
-                it { is_expected.to eq nil }
-              end
+                  describe "#passed" do
+                    subject(:passed) { identity_confirmation_task.passed }
 
-              describe "#manual" do
-                subject(:manual) { identity_confirmation_task.manual }
+                    it { is_expected.to eq nil }
+                  end
 
-                it { is_expected.to eq false }
-              end
-            end
+                  describe "#manual" do
+                    subject(:manual) { identity_confirmation_task.manual }
 
-            describe "note" do
-              subject(:note) { claim_arg.notes.last }
+                    it { is_expected.to eq false }
+                  end
+                end
 
-              before { perform }
+                describe "note" do
+                  subject(:note) { claim_arg.notes.last }
 
-              describe "#body" do
-                subject(:body) { note.body }
+                  before { perform }
 
-                it { is_expected.to eq("National Insurance number not matched") }
-              end
+                  describe "#body" do
+                    subject(:body) { note.body }
 
-              describe "#created_by" do
-                subject(:created_by) { note.created_by }
+                    it do
+                      is_expected.to eq(
+                        <<~HTML
+                          National Insurance number not matched:
+                          <pre>
+                            Claimant: <span class="dark-grey">"</span><span class="red">QQ100000C</span><span class="dark-grey">"</span>
+                            DQT:      <span class="dark-grey">"</span><span class="green">QQ100000B</span><span class="dark-grey">"</span>
```

Comment:
Why not in view or presentation object? Not a big fan of seeing presentation in the model layer